microservices/realtime/pkg/services/redis_helper.go

```diff
@@ -12,15 +12,19 @@ import (
 )
 
 type EventData struct {
-	Event     string `json:"Event"`
-	Time      int64  `json:"Time"`
-	ID        string `json:"ID"`
-	CallID    string `json:"Call-ID"`
-	Party     string `json:"Party"`
-	Brand     string `json:"Brand"`
-	Company   string `json:"Company"`
-	Direction string `json:"Direction"`
-	Owner     string `json:"Owner"`
+	Event       string `json:"Event"`
```

Comment:
Is this indentation style required? Must we update all these lines every time the longest attibute changes?

Reply:
Yes, the indentation is mandatory and generated by `go fmt ./...` command automatically.
Sadly this project lacks a realtime-codestyle check in pipeline :(